src/civic/variant.js

```diff
@@ -297,10 +282,10 @@ const uploadNormalizedVariant = async (conn, normalizedVariant, feature) => {
                 [reference2] = await _entrezGene.fetchAndLoadBySymbol(conn, normalizedVariant.reference2.name);
             }
         }
-        content.reference1 = rid(reference1);
+        content.reference1 = rid(reference1, true);
```

Comment:
reference 1 can never be null

Reply:
Same as here: https://github.com/bcgsc/pori_graphkb_loader/pull/97#discussion_r862027792
Lets roll back and see what append.